### DISCUSSION TITLE
page previews not updating until reopen

### DISCUSSION THREAD

**Discussion by dszgit** [xournalpp/xournalpp#2455] [Q&A] [answered]
I've just found this app, and it looks great!  I built from latest git repo, which went smoothly.  But I have an issue with page previews.  Initially I thought it didn't work at all---the side bar just shows blank pages.  But I've found that when I save and reopen the document, the page previews appear.  They just don't update until I reopen again.  This is under Ubuntu 20.04

I haven't seen any mention of this in Issues, so I figure maybe I'm just missing something.  If not, I will report it, with more system details.  I did see the issue with layer previews not updating until switching between layer and page previews.  But that trick doesn't make my page previews update.

**Comment by Technius** [comment, marked as answer]
I noticed this too, looks like a bug. Please open an issue.

**Comment by rolandlo** [comment]
This has just been fixed in #2630.
